Assets/__Scripts/MapEditor/UI/Bookmarks/BookmarkContainer.cs

```diff
@@ -66,6 +68,24 @@ public void OnPointerUp(PointerEventData eventData)
         }
     }
 
+    public void OnControlModifier(InputAction.CallbackContext context)
+    {
+    }
+
+    public void OnAltModifier(InputAction.CallbackContext context)
+    {
+    }
+
+    public void OnShiftModifier(InputAction.CallbackContext context)
+    {
+        Debug.LogError(context);
+        
+    }
+
+    public void OnMouseMovement(InputAction.CallbackContext context)
+    {
+    }
```

Comment:
Do you mind removing the now unused code since `BookmarkManager` handles the input stuff now?

Reply:
Knew I was forgetting something. Gone now.